### PR TITLE
Fix 65 formatting warnings in botany

### PR DESCRIPTION
**Pull request by slarticodefast**
## About the PR
Project 0 warnings
## Technical details
For better readability I decided to make some of the warnings be ignored using
`#pragma warning disable IDE0055`
`#pragma warning restore IDE0055`

## Media
Im-doing-my-part-meme-9

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
none

**Changelog**
no warnings no fun

### CODE REVIEW

File: Content.Server/Botany/Systems/MutationSystem.cs [hunk 1]
```diff
@@ -155,14 +155,14 @@ private void MutateFloat(ref float val, float min, float max, int bits, int tota
 
         // Starting number of bits that are high, between 0 and bits.
         // In other words, it's val mapped linearly from range [min, max] to range [0, bits], and then rounded.
-        int valInt = (int)MathF.Round((val - min) / (max - min) * bits);
+        int valInt = (int) MathF.Round((val - min) / (max - min) * bits);
```
Comment (Partmedia) [inline Content.Server/Botany/Systems/MutationSystem.cs:158]: Please revert these changes to all the casts, because we follow the [C# convention on casting](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions).

If you can, please investigate why your IDE complains about this.

**Review by Partmedia — CHANGES_REQUESTED** [review]
See comments inline.

File: Content.Server/Botany/Systems/MutationSystem.cs (hunk 1 above)
Comment (slarticodefast) [reply Content.Server/Botany/Systems/MutationSystem.cs:158]: My .editorconfig file is up to date to the [master branch](https://github.com/space-wizards/space-station-14/blob/master/.editorconfig). I think the relevant setting is `csharp_space_after_cast = true` and it should be adjusted if this is unintended. Also most of the code seems to follow that convention, so are you sure it should be changed?

Comment (Partmedia) [reply Content.Server/Botany/Systems/MutationSystem.cs:158]: Yes, this should be adjusted. It is likely this way because of the editor config, and people just follow what their IDE tells them without considering the style guide.

Comment (Partmedia) [reply Content.Server/Botany/Systems/MutationSystem.cs:158]: If you do choose to submit a change to the editorconfig, please make that a separate PR.

Comment (slarticodefast) [reply Content.Server/Botany/Systems/MutationSystem.cs]: Spaces removed. I will make a PR for the editorconfig.

**Review by Partmedia — APPROVED** [review]
